lambdatimes/src/components/Content/Tab.js

```diff
@@ -20,5 +21,8 @@ const Tab = props => {
 };
 
 // Make sure you include PropTypes on your props.
```

Comment:
you can add a couple more, selectTabHandler: PropTypes.func,
selectedTab: PropTypes.string